diffkemp/llvm_ir/build_llvm.py

```diff
@@ -690,6 +691,26 @@ def build_llvm_module(self, name, file_name, commands):
         mod = LlvmKernelModule(name, file_name, self.modules_path)
         return mod
 
+    def build_llvm_function(self, f):
```

Comment:
Better call it build_file_for_function to be consistent with naming of other methods.